content/tutorials/history.textile

```diff
@@ -74,6 +74,24 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/05e2e16
 
+blang[php].
+  To start using Ably you first need to install "composer package on packagist":https://packagist.org/packages/ably/ably-php into your composer.
+
+  ```[sh]
+    composer require ably/ably-php --update-no-dev
+  ```
+
+  The client library must be instanced with the API key you copied in Step 1. API keys used with "basic authentication":/general/authentication#basic-authentication for your own servers is generally preferred, however clients running on insecure devices should always use the "token authentication scheme":/general/authentication#token-authentication instead. In this example, we use an API key for simplicity.
+
+  Add the following to a file named @index.php@ to instance the Ably library inside your php script:
```

Comment:
There is no mention here of `public/index.php` just `index.php`, it should be consistent with the tutorial.